geomesa-core/src/main/scala/org/locationtech/geomesa/core/iterators/TemporalDensityIterator.scala

```diff
@@ -109,6 +115,18 @@ object TemporalDensityIterator extends Logging {
     new Interval(s, e)
   }
 
+  def createFeatureType(origFetType : SimpleFeatureType) = {
```

Comment:
Picky: to be consistent with how we've written most signatures, this should be {parameter: Type} without the space between the parameter and colon. I'd also called this `origFeatureType`